crates/revm/src/inspector/eip3155.rs

```diff
@@ -58,7 +60,7 @@ struct Output {
     error: Option<String>,
     /// Array of all allocated values
     #[serde(default, skip_serializing_if = "Option::is_none")]
-    memory: Option<Vec<String>>,
+    memory: Option<String>,
```

Comment:
Scratch this, it would be better to have `String` here. Conversion is going to slow down the tracer and memory cloning on every step is expensive, but we can't go around this.